StreamChatClient_v3/Database/DatabaseSession_Tests.swift

```diff
@@ -24,14 +24,33 @@ final class DatabaseSession_Tests: XCTestCase {
                                         channel: channelPayload.channel)
         
         // Save the event payload to DB
-        database.write { (session) in
+        database.write { session in
             try session.saveEvent(payload: eventPayload)
         }
         
         // Try to load the saved channel from DB
         var loadedChannel: ChannelModel<DefaultDataTypes>? {
             database.viewContext.loadChannel(cid: channelId)
         }
+        
         AssertAsync.willBeEqual(loadedChannel?.cid, channelId)
+        
+        // Try to load the saved current user from DB
+        if let userId = channelPayload.channel.createdBy?.id {
```

Comment:
In theory, you can you `if let` but then you need to ad `else { XCTFail() }` to fail the test properly if the user is `nil`. In practice, you know the content of the payload, so it's OK to force unwrap here.